lib/ledger_sync/resources/preference.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  class Preference < LedgerSync::Resource
```

Comment:
See my previous comment on the PR, but we should move this under the QBO adaptor and just have it be adaptor-specific so we can define the actual attributes.